### PR TITLE
Use correct grammatical form

### PR DESCRIPTION
**Pull request by peterroth**
Singular if the node count is below 2, plural if more than 1

### CODE REVIEW

**Comment by peterroth** [comment]
Hello @deads2k amd @vrutkovs,
Could you please check this PR? Review was automatically requested but I can see much more activity from you, hence I reach out to you.

File: pkg/operator/staticpod/controller/installer/installer_controller.go [hunk 1]
```diff
@@ -587,7 +587,11 @@ func setAvailableProgressingNodeInstallerFailingConditions(newStatus *operatorv1
 	revisionStrings := []string{}
 	for _, currentRevision := range Int32KeySet(counts).List() {
 		count := counts[currentRevision]
-		revisionStrings = append(revisionStrings, fmt.Sprintf("%d nodes are at revision %d", count, currentRevision))
+		if count > 1 {
```
Comment (vrutkovs) [inline pkg/operator/staticpod/controller/installer/installer_controller.go:590]:
```suggestion
nodesStr = "node"
if count > 1 {
    nodesStr = "nodes"
}
revisionStrings = append(revisionStrings, fmt.Sprintf("%d %s is at revision %d", count, nodesStr currentRevision))
```

**Review by vrutkovs — CHANGES_REQUESTED** [review]

**Review by peterroth — COMMENTED** [review]
The code snippet is modified with the requested change.

**Review by vrutkovs — APPROVED** [review]
/approve
/lgtm

**Review by dgrisonnet — APPROVED** [review]
/approve

**Pull request merged**